src/v0/destinations/ga/networkHandler.js

```diff
@@ -26,16 +30,11 @@ const gaResponseHandler = gaResponse => {
       return errObj.reason && errObj.reason === "invalidCredentials";
     });
     if (isInvalidCredsError || response?.error?.status === "UNAUTHENTICATED") {
-      throw new ErrorBuilder()
-        .setMessage("[GA] invalid credentials")
-        .setStatus(500)
-        .setDestinationResponse(response)
-        .setAuthErrorCategory(REFRESH_TOKEN)
-        .build();
+      throw new RetryableError("invalid credentials");
     }
-    throw new ErrorBuilder()
+    throw new NetworkInstrumentationError()
```

Comment:
1. This is not exactly an instrumentation error.
2. Previous error builder API is used to set other error fields.

Reply:
This is a NetworkError